### ISSUE TITLE
Compilation failed due to missing META-INF

### ISSUE THREAD

**Issue by vovkab** [immutables/immutables#340]
Gradle: 2.10
Android plugin: 2.1.0
Retrolambda: 3.2.0
Immutables: 2.2 (the same on 2.19)

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileRetrolambdaDebug'.
> java.io.FileNotFoundException: my-project/build/intermediates/classes/project/debug/META-INF/services
```

While actual file is generated in: 
```
my-project/build/retrolambda/debug/META-INF/services/com.google.gson.TypeAdapterFactory
```

I'm registering all type adapters manually, so I don't really need services, is there any good way to completely disable generation or maybe there is another way to fix it?

Related issues:
 * TypedAdapterFactory is not loaded with ServiceLoader on Android #100
 * Wrong placement of generated META-INF/services https://github.com/evant/gradle-retrolambda/issues/110

**Comment by elucash** [comment]
1. Trying to not blow up on file not found but issue a warning (may not work as expected, experimental).
2. Knob to disable it `Gson.TypeAdapters.metainfService = false` . `true` by default to enable generation due to backward compatibility concerns.

**Comment by vovkab** [comment]
@elucash Thanks!

**Comment by elucash** [comment]
Released in 2.2.1 Thank you!

**Comment by vovkab** [comment]
Works great, thanks for fix!
